### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by yiahmed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,102 @@
...
+[parameters]: username:string, email:string, password:string, password_validation:string, photo:string
+
+[POST] **/users/signup** Creates a new user
+
+#### User sessions
+[parameters]: username:string, password:string
+
+[POST] **/users/login** Creates a new session
+
+[GET] **/users/logout** Destorys an existing session
```
Comment (Genemichael25) [inline README.md]: Spelling on destroys

File: dependencies.txt
Comment (Genemichael25) [inline dependencies.txt:1]: You really only need the condensed dependencies from package.json and the gemfile.

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: dependencies.txt
Comment (Genemichael25) [inline dependencies.txt:1]: Can we delete this file if we don't need it?

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: dependencies.txt
Comment (yiahmed) [reply dependencies.txt:1]: Group members would like to keep it for future reference

**Review by Genemichael25 — APPROVED** [review]

**Pull request merged**